data/ExperimentDesignPresets.ts

```diff
@@ -0,0 +1,26 @@
+import {AAExperiment} from "../types/experiments";
+
+interface Preset<T> {
+  name: string;
+  description: string;
+  presets: Partial<T>
+}
+
+export const AAType: Preset<AAExperiment> = {
+  name: "A/A Experiment",
+  description: "A design for diagnostic testing of targeting or enrollment. Fixed to 1% of the population.",
+  presets: {
+    branches: [
+      {slug: "control", is_reference_branch: true, ratio: 1, value: {}},
+      {slug: "treatment", ratio: 1, value: {}}
+    ],
```

Comment:
What does this do? Does this fix the number of branches? Is it providing defaults?

Reply:
Because our UI does not expose branch design at the moment, this will fix the number of branches – if there were UI, I think it could be used to populate default values.